vendor/assets/javascripts/dvl/vendor/bootstrap-datepicker.js

```diff
@@ -0,0 +1,1681 @@
+/* =========================================================
+ * bootstrap-datepicker.js
+ * Repo: https://github.com/eternicode/bootstrap-datepicker/
+ * Demo: http://eternicode.github.io/bootstrap-datepicker/
+ * Docs: http://bootstrap-datepicker.readthedocs.org/
+ * Forked from http://www.eyecon.ro/bootstrap-datepicker
+ * =========================================================
+ * Started by Stefan Petre; improvements by Andrew Rowls + contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ========================================================= */
+
+(function($, undefined){
+
+  var $window = $(window);
+
+  function UTCDate(){
+    return new Date(Date.UTC.apply(Date, arguments));
+  }
+  function UTCToday(){
+    var today = new Date();
+    return UTCDate(today.getFullYear(), today.getMonth(), today.getDate());
+  }
+  function alias(method){
+    return function(){
+      return this[method].apply(this, arguments);
+    };
+  }
+
+  var DateArray = (function(){
+    var extras = {
+      get: function(i){
+        return this.slice(i)[0];
+      },
+      contains: function(d){
+        // Array.indexOf is not cross-browser;
+        // $.inArray doesn't work with Dates
+        var val = d && d.valueOf();
+        for (var i=0, l=this.length; i < l; i++)
+          if (this[i].valueOf() === val)
+            return i;
+        return -1;
+      },
+      remove: function(i){
+        this.splice(i,1);
+      },
+      replace: function(new_array){
+        if (!new_array)
+          return;
+        if (!$.isArray(new_array))
+          new_array = [new_array];
```

Comment:
Identifier 'new_array' is not in camel case.<br>Expected '{' and instead saw 'new_array'.